src/server.c

```diff
@@ -4526,7 +4526,7 @@ int finishShutdown(void) {
 
 
     serverLog(LL_WARNING,"%s is now ready to exit, bye bye...",
-        server.sentinel_mode ? "Sentinel" : "Redis");
+        server.sentinel_mode ? "Sentinel" : "Server");
```

Comment:
somehow in here, i prefer Valkey?

Reply:
Agree with @enjoy-binbin. It's like a greeting. Similar to the logo and this one;

    oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo

We can replace that "starting" message with Valkey and different graphics instead of oO0Oo...